src/style.css

```diff
@@ -0,0 +1,453 @@
+}
+
+.add-comment-btn:hover {
+  cursor: pointer;
+  background-color: #ddd;
+}
+
+/* popup styling */
+
+#popup {
+  align-items: center;
+  justify-content: center;
+  display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(0, 0, 0, 0.5);
+  color: #111;
+}
+
+.reservation-container {
+  width: 50%;
+  margin: auto;
+  background-color: #171717;
+  padding: 12px;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  color: #fff;
+}
+
+.poster-container {
+  height: 300px;
+}
+
+.poster-container img {
+  height: 100%;
+}
+
+.details-container {
+  width: 100%;
+}
+
+.details-box {
+  width: 100%;
+  padding: 24px;
+  padding-top: 0;
+  line-height: 1.5;
+  height: 100%;
+}
+
+.details-box ul li {
+  list-style: disc;
+  margin-left: 14px;
+}
+
+.heading {
+  font-size: 32px;
+  padding: 2px 0;
+  border-bottom: 1px solid #ddd;
+  margin-bottom: 14px;
+}
+
+.normal {
+  font-size: 1rem;
+}
+
+.comments {
+  display: grid;
+  grid-template-columns: auto auto;
+  gap: 12px;
+}
+
+.reservations {
+  display: grid;
+  grid-template-columns: auto auto;
+  gap: 0.8rem;
+}
+
+.info-wrapper {
+  display: flex;
+  flex-direction: row;
+}
+
+.close {
+  cursor: pointer;
+  display: inline;
+  width: auto;
+}
+
+.blur {
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100vh;
+  backdrop-filter: blur(10px);
+  z-index: -1;
+}
+
+.forms {
+  background-color: #242424;
+  border-top-left-radius: 9px;
+  border-top-right-radius: 9px;
+  padding: 12px;
+  display: grid;
+  flex-direction: column;
+}
+
+#closeBtn {
+  font-size: x-large;
+  cursor: pointer;
+}
+
+#reserve-button {
+  padding: 10px 20px;
+  background-color: antiquewhite;
+  color: black;
+  border: 1px solid black;
+  border-radius: 10px;
+}
+
+#name,
+#startdate,
+#enddate {
+  height: 1.8rem;
+  margin-bottom: 0.5rem;
+  width: 14rem;
+}
+
+@media only screen and (max-width: 1440px) {
+  #tvShows {
+    grid-template-columns: repeat(3, 30%);
+  }
+}
+
+@media only screen and (max-width: 1024px) {
+  #tvShows {
+    grid-template-columns: repeat(2, 45%);
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  .comments-container {
+    width: 90%;
+  }
+
+  .info-wrapper {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .comments {
+    grid-template-columns: auto;
+  }
+
+  .forms {
+    order: 1;
+  }
+
+  .reservation-container {
+    width: 90%;
+  }
+
+  .reservations {
+    grid-template-columns: auto;
+  }
+
+  #popup {
+    padding-top: 0;
+  }
+}
+
+@media only screen and (max-width: 680px) {
+  #tvShows {
+    grid-template-columns: repeat(1, 90%);
+  }
+}
+
+@media only screen and (min-width: 430px) {
+  #hamburguer {
+    display: none;
+  }
+}
+
+@media only screen and (max-width: 425px) {
+  header {
+    display: none;
+  }
+
+  #hamburguer.hide {
+    display: block;
+  }
+
+  #hamburguer {
+    display: none;
+  }
```

Comment:
- [ ] Kindly avoid code repetition in order to follow CSS best practises. 👍 You nicely did the needfull in line 374. 👏

Reply:
@OyePriscilla  these css codes are for different breakpoints of various screens, they are not repetitions.

Reply:
Done 💯